Source/Extensions/Blazorise.Splitter/wwwroot/blazorise.splitter.js

```diff
@@ -0,0 +1,81 @@
+import './vendors/split.js?v=1.1.3.0';
+
+document.getElementsByTagName("head")[0].insertAdjacentHTML("beforeend", "<link rel=\"stylesheet\" href=\"_content/Blazorise.Splitter/blazorise.splitter.css?v=1.1.3.0\" />");
```

Comment:
We do, actually. For example, on Animate, Cropper, Markown, Video, etc.

Reply:
I see, my bad then, I spoke from memory. I was thinking if users have static files being served from elsewhere, this will probably fail. Anyway they can still load it manually.